.ado/publish.yml

```diff
@@ -107,7 +107,7 @@ jobs:
   - job: RNMacOSInitNpmJSPublish
     displayName: react-native-macos-init Publish to npmjs.org
     pool:
-      vmImage: vs2017-win2016
+      vmImage: $(VmImageWindows)
```

Comment:
I do a similar change in https://github.com/microsoft/react-native-macos/pull/942 , so there might be a merge conflict pending which change goes in first (probably yours)

Reply:
yeah makes sense - lmk how that goes 👍

Reply:
I just merged my change. We should no longer have any windows VMs in our ADO pipeline =)